src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/service/ServiceCategoryService.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.service
+
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceCategory
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository.ServiceCategoryRepository
+import java.util.UUID
+
+@Service
+class ServiceCategoryService(val repository: ServiceCategoryRepository) {
+
+  fun getServiceCategory(id: UUID): ServiceCategory? {
```

Comment:
nitpick: prefer `getServiceCategoryByID` or similar

Reply:
No worries - happy to change